static/package.json

```diff
@@ -52,9 +51,11 @@
     "iframe-resizer": "^4.1.1",
     "is-mobile": "^2.2.2",
     "libphonenumber-js": "^1.7.51",
+    "lodash": "^4.17.20",
```

Comment:
Kinda a cheat move, but I think you can put this in the `package.json`, run `npm install` to get the updates in the `package-lock`, then remove this from the `package.json`. I think that will get you what you want (updating the transitive lodash dependencies), without making lodash a top-level dependency.

Reply:
nifty!